docs/modules/ROOT/pages/references/service-maturity.adoc

```diff
@@ -0,0 +1,294 @@
+With "feature complete" it's not meant that the service doesn't receive new functionality or doesn't get maintained anymore, we're talking about the default scope outlined on this page.
+
+Important is that every step is independent of each other. A service that implements `Backup and Restore` doesn't necessarily already implement `Logs`.
+That in turn also means we must allow some leeway for each service in respect to the requirements, should a service not allow for a feature or makes it overly complex.
+A service is allowed to deviate from a requirement when it makes sense and can be documented and reasoned about accordingly.
+
+TODO! To understand the personas referenced in this document, see xref:reference/glossary.adoc#_personas[Glossary - Personas].
+
+TODO! We currently maintain the service maturity assessment for each service on our product documentation site under https://products.docs.vshn.ch/products/appuio/managed/services_index.html[VSHN Application Catalog -> Services^].
+
+== Initial Release: Cluster Provisioning
+
+As a user of the service I can perform the following actions:
+
+Provisioning::
+I can provision a production-ready cloud Kubernetes environment on a major cloud service provider, such as Amazon EKS, Azure AKS, Google GKE, Exsoscale SKS (on request) and Swisscom ESC (on request).
+
+Scalability::
+I can easily scale my cluster as my needs change, without any downtime or disruption to my services.
+
+Customization::
+I can customize my cluster with the features and components I need to run my workloads.
+
+Security::
+The service is built with security in mind and includes features such as encrypted communication, role-based access control (RBAC), and more to help me protect my data and workloads.
+
+Technical support::
+I can receive technical support from Kubernetes experts, who can help me resolve any issues or questions I may have, according to my SLA.
+
+
+== Iteration: Cluster Maintainence
+
+As a user of the service I can perform the following actions:
+
+Scheduling::
+I can schedule maintenance windows for my cloud Kubernetes environment, during which updates, patches, or other changes can be applied automatically.
+
+Notifications::
+I can receive notifications in advance of any scheduled maintenance, allowing me to plan accordingly and to minimize any disruption to my applications.
+
+Automated Updates::
+The service will automatically apply updates and patches to my cloud Kubernetes environment, ensuring that it is always running the latest and most secure version of the software.
+
+Monitoring::
+I can monitor the status of my cloud Kubernetes environment during the maintenance window, to ensure that the maintenance is operating as expected.
+
+
+== Iteration: Cluster Upgrade
+
+As a user of the service I can perform the following actions:
+
+Upgrade::
+I can upgrade my cloud Kubernetes clusters minor version.
```

Comment:
```suggestion
I can upgrade my Cloud Kubernetes clusters minor version.
```

Maybe rephrase into "I can schedule an automated upgrade of my Cloud Kubernetes cluster to the next minor version".